talos/manipulate_boxes/common_hpp.py

```diff
@@ -817,6 +823,5 @@ def addCostToComponent(graph, costs, state=None, edge=None):
     _cp = _configConstraint.getConfigProjector()
     _cp.setLastIsOptional(True)
     for cost in costs:
-        _cost = graph.clientBasic.problem.getConstraint(cost)                                                                                                                                                 
-        _cp.add(_cost, 1) 
-
+        _cost = graph.clientBasic.problem.getConstraint(cost)
+        _cp.add(_cost, 1)
```

Comment:
I don't understand this change.

Reply:
I think it is white space deletion. I can revert if you want.

Reply:
No, that's fine.